frame/contracts/src/exec.rs

```diff
@@ -1015,6 +1019,20 @@ where
 		self.run(executable, input_data)
 	}
 
+	fn set_code_hash(&mut self, hash: CodeHash<Self::T>) -> Result<(), DispatchError> {
+		increment_refcount::<Self::T>(hash)?;
```

Comment:
Is the turbo fish here really necessary? Looks like it should be able to be inferred from the argument. Same for the other occurrences in this function.